helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/ClusterModelProvider.java

```diff
@@ -125,11 +125,9 @@ public static ClusterModel generateClusterModel(ResourceControllerDataProvider d
         // 2. if the resource does appear in the best possible assignment, need to reassign.
         if (clusterChanges
             .getOrDefault(HelixConstants.ChangeType.RESOURCE_CONFIG, Collections.emptySet())
-            .contains(resourceName)
-            || clusterChanges
+            .contains(resourceName) || clusterChanges
```

Comment:
Why is this changing? Was this not formatted before?

Reply:
I think I manually formatted because of some comments. Anyway, the current version is automatically formatted.